src/quacc/calculators/espresso/keys.py

```diff
@@ -0,0 +1,168 @@
+# We work ase.io.espresso style, but we need to add the namelist keys for each binary.
+# !!! For now this is GPT3.5 generated, must be checked !!!
+
+from ase.io.espresso import Namelist
+
+PH_KEYS = Namelist([
```

Comment:
Would you be able to briefly explain what the purpose is of this namelist stuff? Thanks. I trust it makes sense but haven't dug through the ASE docs for this yet. [Of course, if there are ways we can avoid direct enumeration of things, that'd be great, but I imagine you are well-aware of that].